controller.go

```diff
@@ -135,6 +137,22 @@ func (c *Controller) RenderTemplate(templatePath string) Result {
 	}
 }
 
+//receive a post method of json request.
```

Comment:
// ReceiveJSON unmarshal the input streaming JSON data, returns `map[string]interface{}` type data.
// To access this data we can use a type assertion to access map[string]interface{}. 
// Reference to [Decoding arbitrary data chapter] in https://blog.golang.org/json-and-go